assembly/dependencies-apache-ignite.xml

```diff
@@ -130,7 +130,6 @@
                 <exclude>org.apache.ignite:ignite-hibernate-core</exclude>
                 <exclude>org.apache.ignite:ignite-hibernate_4.2</exclude>
                 <exclude>org.apache.ignite:ignite-hibernate_5.1</exclude>
-                <exclude>org.apache.ignite:ignite-schedule</exclude>
```

Comment:
Why?

Reply:
It is needed to add ignite-schedule module to nolgpl buid. Peter Ivanov suggested this change.